From ec2c8215a583f7d86846c4fa988766380d5990ad Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinh.nguyen@intel.com>
Date: Fri, 20 Oct 2017 18:36:15 -0500
Subject: [PATCH 01/25] arm64: dts: stratix10: Use PSCI spin-table for SMP

commit  bf2f49493b4e7d8783171d7ba1e1f24bed921594 from
https://github.com/altera-opensource/linux-socfpga.git

This reverts commit ba759bf25bc5 ("FogBugz #488851: dts: use
 spin-table for SMP")
U-Boot has implemented PSCI which is preferable to the spin-table
so revert the temporary spin-table changes.

Signed-off-by: Dinh Nguyen <dinguyen@kernel.org>
Signed-off-by: Thor Thayer <thor.thayer@linux.intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi b/arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi
index fb0698a..8ea5dae 100644
--- a/arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi
+++ b/arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi
@@ -31,28 +31,28 @@
 		cpu0: cpu@0 {
 			compatible = "arm,cortex-a53", "arm,armv8";
 			device_type = "cpu";
-			enable-method = "spin-table";
+			enable-method = "psci";
 			reg = <0x0>;
 		};
 
 		cpu1: cpu@1 {
 			compatible = "arm,cortex-a53", "arm,armv8";
 			device_type = "cpu";
-			enable-method = "spin-table";
+			enable-method = "psci";
 			reg = <0x1>;
 		};
 
 		cpu2: cpu@2 {
 			compatible = "arm,cortex-a53", "arm,armv8";
 			device_type = "cpu";
-			enable-method = "spin-table";
+			enable-method = "psci";
 			reg = <0x2>;
 		};
 
 		cpu3: cpu@3 {
 			compatible = "arm,cortex-a53", "arm,armv8";
 			device_type = "cpu";
-			enable-method = "spin-table";
+			enable-method = "psci";
 			reg = <0x3>;
 		};
 	};
-- 
2.8.1

